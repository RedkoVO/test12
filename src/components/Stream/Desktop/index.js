import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import MenuNavigation from '../../MenuNavigation'
import HeaderCatalog from '../components/HeaderCatalog'
import Footer from '../../Footer/Desktop'

import LeftTeamLogo from '../../../assets/images/leftTeamLogo.png'
import LeftTeamFlag from '../../../assets/images/leftTeamFlag.png'
import RightTeamLogo from '../../../assets/images/rightTeamLogo.png'
import RightTeamFlag from '../../../assets/images/rightTeamFlag.png'
import StreamEvent from '../../../assets/images/streamEvent.png'
import StreamImage from '../../../assets/images/stream-image.png'
import TmpStreamTabsImage from '../../../assets/images/tmp-stream-tabs.png'

import styles from './styles'

const Stream = ({ classes, handleLogout, isDesktop }) => (
  <div className="containerStreamWrap">
    <div className={classes.containerTradeWrap}>
      <div className={classes.containerTrade}>
        <header className={classes.headerStream}>
          <MenuNavigation page="dark" handleLogout={handleLogout} />
          <HeaderCatalog isDesktop={isDesktop} />
        </header>

        <main className="streamMain">
          <div className="streamBettingTop">
            <div className="streamLeftTeam">
              <div>
                <div className="streamTeamName">
                  <img src={LeftTeamLogo} className="streamLogo" alt="" />
                  NRG
                </div>
                <div className="streamCountry">
                  <img src={LeftTeamFlag} className="streamFlag" alt="" />
                  France
                </div>
              </div>
            </div>

            <div className="streamCoef">
              <div className="streamCoefItem">6.17</div>
              <div className="streamVersus">vs</div>
              <div className="streamCoefItem">1.25</div>
            </div>

            <div className="streamRightTeam">
              <div className="streamRightTeam">
                <div>
                  <div className="streamTeamName">
                    <img src={RightTeamLogo} className="streamLogo" alt="" />
                    LDLC
                  </div>
                  <div className="streamCountry">
                    <img src={RightTeamFlag} className="streamFlag" alt="" />
                    Spain
                  </div>
                </div>
              </div>
              <div className="streamLive">Live</div>
            </div>

            <div className="streamEvent">
              Event:
              <img src={StreamEvent} />
            </div>
          </div>

          <div className="streamContent">
            <img src={StreamImage} alt="" />
          </div>

          <div className="streamDescription">
            <div className="streamDescriptionLeft">
              <div className="streamDescriptionText">
                Integer tempor semper sapien sit amet bibendum. Etiam ornare
                tellus at sapien vehicula, quis scelerisque turpis rokaste
                egestas quis botona.
              </div>
              <div className="streamHashTags">
                #ROLLED #BESTCRYPTO #ETH{' '}
                <div className="streamAddHashTags">+</div>
              </div>
            </div>
            <div className="streamDescriptionRight">
              <div className="streamAddFavorites">Add to favorites</div>
              <div className="streamShareSocial">Share in social</div>
            </div>
          </div>

          <div className="streamTmpTabs">
            <img src={TmpStreamTabsImage} alt="" />
          </div>
        </main>
      </div>
    </div>
    <Footer />
  </div>
)

Stream.propTypes = {
  classes: PropTypes.object,
  handleLogout: PropTypes.func,
  isDesktop: PropTypes.bool
}

export default withStyles(styles)(Stream)
