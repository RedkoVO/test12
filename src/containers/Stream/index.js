import compose from 'recompose/compose'
import { pure } from 'recompose'

import AsyncStreamDesktop from '../../components/Stream/Desktop/AsyncStreamDesktop'

export default compose(
  pure
)(AsyncStreamDesktop)
