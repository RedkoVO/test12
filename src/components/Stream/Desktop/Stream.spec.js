import * as React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'

import Stream from './'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter keyLength={0}>
        <Stream handleLogout={() => {}} isDesktop={false} />
      </MemoryRouter>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
