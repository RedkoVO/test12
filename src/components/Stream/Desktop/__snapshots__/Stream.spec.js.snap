// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="containerStreamWrap"
>
  <div
    className="Stream-containerTradeWrap-2"
  >
    <div
      className="Stream-containerTrade-3"
    >
      <header
        className="Stream-headerStream-1"
      >
        <ul
          className="MenuNavigation-root-5 dark"
        >
          <li
            className={undefined}
          >
            <div
              className="MenuNavigation-logout-14"
              onClick={[Function]}
            >
              Logout
            </div>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-dashboard-8"
              />
              <br />
               Dashboard
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-walletIcon-9"
              />
              <br />
               Wallet
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-shop-10"
              />
              <br />
               Shop
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/games"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-games-11"
              />
              <br />
               Games
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-friends-12"
              />
              <br />
               Friends
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-settings-13"
              />
              <br />
               Settings
            </a>
          </li>
        </ul>
      </header>
      <main
        className="Stream-streamMain-4"
      >
        <div
          className="StreamBettingTop-root-15"
        >
          <div
            className="StreamBettingTop-streamLeftTeam-16"
          >
            <div>
              <div
                className="StreamBettingTop-streamTeamName-18"
              >
                <img
                  alt=""
                  className="StreamBettingTop-streamLeftLogo-17"
                  src="leftTeamLogo.png"
                />
                NRG
              </div>
              <div
                className="StreamBettingTop-streamCountry-19"
              >
                <img
                  alt=""
                  className="StreamBettingTop-streamFlag-20"
                  src="leftTeamFlag.png"
                />
                France
              </div>
            </div>
          </div>
          <div
            className="StreamBettingTop-streamCoef-21"
          >
            <div
              className="StreamBettingTop-streamCoefItem-22"
            >
              6.17
            </div>
            <div
              className="StreamBettingTop-streamVersus-23"
            >
              vs
            </div>
            <div
              className="StreamBettingTop-streamCoefItem-22"
            >
              1.25
            </div>
          </div>
          <div
            className="StreamBettingTop-streamRightTeam-24"
          >
            <div
              className="StreamBettingTop-streamRightTeam-24"
            >
              <div>
                <div
                  className="StreamBettingTop-streamTeamName-18"
                >
                  <img
                    alt=""
                    className="StreamBettingTop-streamRightLogo-25"
                    src="rightTeamLogo.png"
                  />
                  LDLC
                </div>
                <div
                  className="StreamBettingTop-streamCountry-19"
                >
                  <img
                    alt=""
                    className="StreamBettingTop-streamFlag-20"
                    src="rightTeamFlag.png"
                  />
                  Spain
                </div>
              </div>
            </div>
            <div
              className="StreamBettingTop-streamLive-26"
            >
              Live
            </div>
          </div>
          <div
            className="StreamBettingTop-streamEvent-27"
          >
            Event:
            <img
              alt=""
              src="streamEvent.png"
            />
          </div>
        </div>
        <div
          className="StreamScreen-root-28"
        >
          <img
            alt=""
            src="stream-image.png"
          />
        </div>
        <div
          className="StreamDescription-root-29"
        >
          <div
            className="StreamDescription-streamDescriptionLeft-30"
          >
            <div
              className="StreamDescription-streamDescriptionText-31"
            >
              Integer tempor semper sapien sit amet bibendum. Etiam ornare tellus at sapien vehicula, quis scelerisque turpis rokaste egestas quis botona.
            </div>
            <div
              className="StreamDescription-streamHashTags-32"
            >
              #ROLLED #BESTCRYPTO #ETH
               
              <div
                className="StreamDescription-streamAddHashTags-33"
              >
                +
              </div>
            </div>
          </div>
          <div
            className="StreamDescription-streamDescriptionRight-34"
          >
            <div
              className="StreamDescription-streamAddFavorites-35"
            >
              Add to favorites
            </div>
            <div
              className="StreamDescription-streamShareSocial-36"
            >
              Share in social
            </div>
          </div>
        </div>
        <div
          className="StreamTabs-root-37"
        >
          <div
            className="StreamTabs-streamTabsTop-38"
          >
            <div
              className="StreamTabs-streamTabsUpcom-39"
              href="#"
            >
              UPCOMING GAMES
            </div>
            <div
              className="StreamTabs-streamTabsResult-40"
              href="#"
            >
              RESULTS
            </div>
          </div>
          <div
            className="StreamTabsBottom-root-41"
          >
            <div
              className="StreamTabsNav-root-42"
            >
              <div
                className="StreamTabsNav-streamTabsNavItem-43 StreamTabsNav-liveNow-44"
              >
                LIVE NOW 
                <span
                  className={undefined}
                >
                  2
                </span>
              </div>
              <div
                className="StreamTabsNav-streamTabsNavItem-43 StreamTabsNav-allGames-45"
              >
                ALL GAMES 
                <span
                  className={undefined}
                >
                  2
                </span>
              </div>
              <div
                className="StreamTabsNav-streamTabsNavItem-43 StreamTabsNav-dota2-46"
              >
                DOTA 2
                <span
                  className={undefined}
                >
                  2
                </span>
              </div>
              <div
                className="StreamTabsNav-streamTabsNavItem-43 StreamTabsNav-starcraft2-47"
              >
                STARCRAFT 2 
                <span
                  className={undefined}
                >
                  2
                </span>
              </div>
              <div
                className="StreamTabsNav-streamTabsNavItem-43 StreamTabsNav-overwatch-48"
              >
                OVERWATCH 
                <span
                  className={undefined}
                >
                  2
                </span>
              </div>
              <div
                className="StreamTabsNav-streamTabsNavItem-43 StreamTabsNav-rainbowSix-49"
              >
                RAINBOW SIX 
                <span
                  className={undefined}
                >
                  2
                </span>
              </div>
              <div
                className="StreamTabsNav-streamTabsNavItem-43 StreamTabsNav-csGo-51 StreamTabsNav-streamTabsNavActive-50"
              >
                CS:GO
                <span
                  className={undefined}
                >
                  2
                </span>
              </div>
            </div>
            <div
              className="StreamLive-root-52"
            >
              <div
                className={undefined}
              >
                <div
                  className=""
                >
                  Live
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    NRG
                  </div>
                  <div
                    className=""
                  >
                    France
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  6.14
                </div>
                <div
                  className={undefined}
                >
                  VS
                </div>
                <div
                  className={undefined}
                >
                  1.25
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    LDLC
                  </div>
                  <div
                    className=""
                  >
                    Spain
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  EVENT
                </div>
                <div
                  className={undefined}
                />
              </div>
              <div
                className={undefined}
              >
                <div
                  className=""
                >
                  12:30 PM
                   
                  <br />
                  <span
                    className="dateDay"
                  >
                    7/12/2018
                  </span>
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    NRG
                  </div>
                  <div
                    className=""
                  >
                    France
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  6.14
                </div>
                <div
                  className={undefined}
                >
                  VS
                </div>
                <div
                  className={undefined}
                >
                  1.25
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    LDLC
                  </div>
                  <div
                    className=""
                  >
                    Spain
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  EVENT
                </div>
                <div
                  className={undefined}
                />
              </div>
              <div
                className={undefined}
              >
                <div
                  className=""
                >
                  Live
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    NRG
                  </div>
                  <div
                    className=""
                  >
                    France
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  6.14
                </div>
                <div
                  className={undefined}
                >
                  VS
                </div>
                <div
                  className={undefined}
                >
                  1.25
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    LDLC
                  </div>
                  <div
                    className=""
                  >
                    Spain
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  EVENT
                </div>
                <div
                  className={undefined}
                />
              </div>
              <div
                className={undefined}
              >
                <div
                  className=""
                >
                  12:30 PM
                   
                  <br />
                  <span
                    className="dateDay"
                  >
                    7/12/2018
                  </span>
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    NRG
                  </div>
                  <div
                    className=""
                  >
                    France
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  6.14
                </div>
                <div
                  className={undefined}
                >
                  VS
                </div>
                <div
                  className={undefined}
                >
                  1.25
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    LDLC
                  </div>
                  <div
                    className=""
                  >
                    Spain
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  EVENT
                </div>
                <div
                  className={undefined}
                />
              </div>
              <div
                className={undefined}
              >
                <div
                  className=""
                >
                  Live
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    NRG
                  </div>
                  <div
                    className=""
                  >
                    France
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  6.14
                </div>
                <div
                  className={undefined}
                >
                  VS
                </div>
                <div
                  className={undefined}
                >
                  1.25
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    LDLC
                  </div>
                  <div
                    className=""
                  >
                    Spain
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  EVENT
                </div>
                <div
                  className={undefined}
                />
              </div>
              <div
                className={undefined}
              >
                <div
                  className=""
                >
                  12:30 PM
                   
                  <br />
                  <span
                    className="dateDay"
                  >
                    7/12/2018
                  </span>
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    NRG
                  </div>
                  <div
                    className=""
                  >
                    France
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  6.14
                </div>
                <div
                  className={undefined}
                >
                  VS
                </div>
                <div
                  className={undefined}
                >
                  1.25
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    LDLC
                  </div>
                  <div
                    className=""
                  >
                    Spain
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  EVENT
                </div>
                <div
                  className={undefined}
                />
              </div>
              <div
                className={undefined}
              >
                <div
                  className=""
                >
                  Live
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    NRG
                  </div>
                  <div
                    className=""
                  >
                    France
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  6.14
                </div>
                <div
                  className={undefined}
                >
                  VS
                </div>
                <div
                  className={undefined}
                >
                  1.25
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    LDLC
                  </div>
                  <div
                    className=""
                  >
                    Spain
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  EVENT
                </div>
                <div
                  className={undefined}
                />
              </div>
              <div
                className={undefined}
              >
                <div
                  className=""
                >
                  12:30 PM
                   
                  <br />
                  <span
                    className="dateDay"
                  >
                    7/12/2018
                  </span>
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    NRG
                  </div>
                  <div
                    className=""
                  >
                    France
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  6.14
                </div>
                <div
                  className={undefined}
                >
                  VS
                </div>
                <div
                  className={undefined}
                >
                  1.25
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    LDLC
                  </div>
                  <div
                    className=""
                  >
                    Spain
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  EVENT
                </div>
                <div
                  className={undefined}
                />
              </div>
              <div
                className={undefined}
              >
                <div
                  className=""
                >
                  Live
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    NRG
                  </div>
                  <div
                    className=""
                  >
                    France
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  6.14
                </div>
                <div
                  className={undefined}
                >
                  VS
                </div>
                <div
                  className={undefined}
                >
                  1.25
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    LDLC
                  </div>
                  <div
                    className=""
                  >
                    Spain
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  EVENT
                </div>
                <div
                  className={undefined}
                />
              </div>
              <div
                className={undefined}
              >
                <div
                  className=""
                >
                  12:30 PM
                   
                  <br />
                  <span
                    className="dateDay"
                  >
                    7/12/2018
                  </span>
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    NRG
                  </div>
                  <div
                    className=""
                  >
                    France
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  6.14
                </div>
                <div
                  className={undefined}
                >
                  VS
                </div>
                <div
                  className={undefined}
                >
                  1.25
                </div>
                <div
                  className=""
                >
                  <div
                    className={undefined}
                  >
                    LDLC
                  </div>
                  <div
                    className=""
                  >
                    Spain
                  </div>
                </div>
                <div
                  className={undefined}
                >
                  EVENT
                </div>
                <div
                  className={undefined}
                />
              </div>
            </div>
          </div>
        </div>
      </main>
    </div>
  </div>
  <footer
    className="Footer-root-53"
  >
    <ul
      className="Footer-footerNav-54"
    >
      <li>
        <a
          className="Footer-footerNavItem-55"
          href="/"
          onClick={[Function]}
        >
          Dashboard
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-55"
          href="/"
          onClick={[Function]}
        >
          Wallet
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-55"
          href="/"
          onClick={[Function]}
        >
          Shop
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-55"
          href="/games"
          onClick={[Function]}
        >
          Games
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-55"
          href="/"
          onClick={[Function]}
        >
          Friends
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-55"
          href="/"
          onClick={[Function]}
        >
          Settings
        </a>
      </li>
    </ul>
    <span
      className={undefined}
    >
      Vault. All Rights Reserved 2018
    </span>
  </footer>
</div>
`;
